src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+import java.util.ArrayList;
+
+/**
+ * TaskList utitlises an ArrayList to hold it's tasks.
+ */
+public class TaskList {
+    ArrayList<Task> list;
```

Comment:
It might be better to use a List interface in case you decide to use a different list in the future.